Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM scratch AS standalone-binary
+
+# @TODO change to --chmod=+x when https://github.com/moby/buildkit/pull/5380 is released
```

Comment:
Tracking this todo; looks like the PR is merged, I have no idea if it was released/usable yet though

Reply:
Yes it's merged and I even tested this at some point, but it wasn't working - seems like it's not released yet, hence the `@TODO`.

Reply:
It was released under [experimental `1.11.0-labs` tag](https://github.com/moby/buildkit/releases/tag/dockerfile/1.11.0-labs) and can be used as opt-in by adding `# syntax=docker.io/docker/dockerfile:1.11.0-labs` to the Dockerfile. Do we want to add it, or just update it when it's released in stable version?

Reply:
There is a newer release 1.12 now, is this possible to use with a stable version?

Reply:
1.12 does not contain this. I just tested labs release

```Dockerfile
# syntax=docker.io/docker/dockerfile:1.11.0-labs

FROM scratch AS standalone-binary

COPY --chmod=+x pie.phar /pie
```

But it does not work somehow 🤷:

```
$  docker build --progress=plain --no-cache -t pie-local .
#0 building with "orbstack" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 152B done
#1 DONE 0.0s

#2 resolve image config for docker-image://docker.io/docker/dockerfile:1.11.0-labs
#2 ...

#3 [auth] docker/dockerfile:pull token for registry-1.docker.io
#3 DONE 0.0s

#2 resolve image config for docker-image://docker.io/docker/dockerfile:1.11.0-labs
#2 DONE 1.9s

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 152B done
#1 DONE 0.0s

#4 docker-image://docker.io/docker/dockerfile:1.11.0-labs@sha256:51bca3eb9dfe8ad06ee1b137d3fa44f3cef79200d5b0010b580da63d128609e4
#4 CACHED

#5 [internal] load build definition from Dockerfile
#5 DONE 0.0s

#6 [internal] load .dockerignore
#6 transferring context: 2B done
#6 DONE 0.0s
Dockerfile:5
--------------------
   3 |     FROM scratch AS standalone-binary
   4 |     
   5 | >>> COPY --chmod=+x pie.phar /pie
   6 |     
--------------------
ERROR: failed to solve: invalid chmod parameter: '+x'. it should be octal string and between 0 and 07777
```

Let me know if you want to keep `@TODO` or just stick with currently used `--chmod=0755` (remove the comment), which works correctly.

Reply:
OK lets leave as-is for now